MvvmCross_All.sln

```diff
@@ -407,13 +407,27 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MvvmCross.Plugins.Visibilit
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MvvmCross.Plugins.WebBrowser.Uwp", "MvvmCross-Plugins\WebBrowser\MvvmCross.Plugins.WebBrowser.Uwp\MvvmCross.Plugins.WebBrowser.Uwp.csproj", "{08E19EF4-8C86-4F6E-B78A-063983884434}"
 EndProject
-Project("{9344BDBB-3E7F-41FC-A0DD-8665D75EE146}") = "MvvmCross.Platform.Wpf", "MvvmCross\Platform\Wpf\MvvmCross.Platform.Wpf.csproj", "{BDD29D7B-293A-4A4A-8DC8-F18285CBFD72}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MvvmCross.Platform.Wpf", "MvvmCross\Platform\Wpf\MvvmCross.Platform.Wpf.csproj", "{3688D6FC-E7F4-485F-8B27-96C0A8C7EAED}"
```

Comment:
After taking a look at it, I think we should accept the id change - if you edit the MvvmCross_All.sln , you'll notice that all projects are using the 'FAE04EC0-301F-11D3-BF4B-00C04F79EFBC' id.

It seems that MvvmCross.Platform.Wpf was the only one using the '9344BDBB-3E7F-41FC-A0DD-8665D75EE146' id - and somehow VS figured it out and fixed it.